ocs_ci/ocs/resources/ocs.py

```diff
@@ -109,6 +110,15 @@ def delete(self, wait=True, force=False):
             bool: True if deleted, False otherwise
 
         """
+        # Avoid accidental delete of default storageclass and secret
```

Comment:
This is the API layer of OCS.
It shouldn't include such terms. 
We need to handle it in the fixtures (factory level). 

If we have tests that deletes a storage class or secret as part of the test, those tests shouldn't use the default

Reply:
OCS is a our storage object, So I think it is fine to handle delete Defaults OCS SCs that are accidentally deleted by tests. In my fixture we are not deleting it, but I am have seen one or two tests trying to remove them and its better to handle them.